classes/PodsMeta.php

```diff
@@ -1494,17 +1489,24 @@ public function meta_media( $form_fields, $post ) {
 
 				pods_no_conflict_off( 'post' );
 
-				// Manually force DFV initialization.  This is needed for attachments in "grid mode" in the
-				// media library.  Note that this should only occur for attachment_fields_to_edit (see #4785)
-				$dfv_init_script = "<script>window.PodsDFV.init();</script>";
-
 				$form_fields[ 'pods_meta_' . $field['name'] ] = array(
 					'label' => $field['label'],
 					'input' => 'html',
-					'html'  => PodsForm::field( 'pods_meta_' . $field['name'], $value, $field['type'], $field, $pod, $id ) . $meta_nonce . $dfv_init_script,
+					'html'  => PodsForm::field( 'pods_meta_' . $field['name'], $value, $field['type'], $field, $pod, $id ),
 					'helps' => PodsForm::comment( 'pods_meta_' . $field['name'], $field['description'], $field )
 				);
 
+				// Manually force DFV initialization.  This is needed for attachments in "grid mode" in the
+				// media library.  Note that this should only occur for attachment_fields_to_edit (see #4785)
+				$dfv_init_script = "<script>window.PodsDFV.init();</script>";
+
+				// @todo is there a cleaner way to make sure these aren't repeated for
+				// each field on the page?
```

Comment:
No cleaner way really, this is the best way we have available to us :/

```suggestion
				// Only output nonce/init script on the very first field of the first group we have.
```